src/Tribe/Admin/Help_Page.php

```diff
@@ -0,0 +1,448 @@
+ */
+class Tribe__Admin__Help_Page {
+	/**
+	 * Static Singleton Holder
+	 * @var Tribe__Admin__Helpers|null
+	 */
+	protected static $instance;
+
+	/**
+	 * Static Singleton Factory Method
+	 *
+	 * @return Tribe__Admin__Helpers
+	 */
+	public static function instance() {
+		if ( ! isset( self::$instance ) ) {
+			$className      = __CLASS__;
+			self::$instance = new $className;
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Get the list of plugins
+	 *
+	 * @param  string  $plugin    Should get only one plugin?
+	 * @param  boolean $is_active Only get active plugins?
+	 * @return array
+	 */
+	public function get_plugins( $plugin = null, $is_active = true ) {
+		$plugins = array();
+
+		$plugins['the-events-calendar'] = array(
+			'name' => 'the-events-calendar',
+			'title' => __( 'The Events Calendar', 'tribe-common' ),
+			'api' => 'http://wpapi.org/api/plugin/the-events-calendar.php',
+			'repo' => 'http://wordpress.org/extend/plugins/the-events-calendar/',
+			'stars_url' => 'http://wordpress.org/support/view/plugin-reviews/the-events-calendar?filter=5',
+			'description' => __( 'The Events Calendar is a carefully crafted, extensible plugin that lets you easily share your events.', 'tribe-common' ),
+			'is_active' => false,
+			'version' => null,
+		);
+
+		if ( class_exists( 'Tribe__Events__Main' ) ){
+			$plugins['the-events-calendar']['version'] = Tribe__Events__Main::VERSION;
+			$plugins['the-events-calendar']['is_active'] = true;
+		}
+
+		$plugins['event-tickets'] = array(
+			'name' => 'event-tickets',
+			'title' => __( 'Event Tickets', 'tribe-common' ),
+			'api' => 'http://wpapi.org/api/plugin/event-tickets.php',
+			'repo' => 'http://wordpress.org/extend/plugins/event-tickets/',
+			'stars_url' => 'http://wordpress.org/support/view/plugin-reviews/event-tickets?filter=5',
+			'description' => __( 'Events Tickets is a carefully crafted, extensible plugin that lets you easily sell tickets for your events.', 'tribe-common' ),
+			'is_active' => false,
+			'version' => null,
+		);
+
+		if ( class_exists( 'Tribe__Tickets__Main' ) ){
+			$plugins['event-tickets']['version'] = Tribe__Tickets__Main::VERSION;
+			$plugins['event-tickets']['is_active'] = true;
+		}
+
+		$plugins['advanced-post-manager'] = array(
+			'name' => 'advanced-post-manager',
+			'title' => __( 'Advanced Post Manager', 'tribe-common' ),
+			'api' => 'http://wpapi.org/api/plugin/advanced-post-manager.php',
+			'repo' => 'http://wordpress.org/extend/plugins/advanced-post-manager/',
+			'stars_url' => 'http://wordpress.org/support/view/plugin-reviews/advanced-post-manager?filter=5',
+			'description' => null,
+			'is_active' => false,
+			'version' => null,
+		);
+
+		if ( class_exists( 'Tribe_APM' ) ){
+			$plugins['advanced-post-manager']['is_active'] = true;
+		}
+
+		$plugins = (array) apply_filters( 'tribe_help_plugins', $plugins );
+
+		// Only active ones?
+		if ( true === $is_active ){
+			foreach ( $plugins as $key => $plugin ) {
```

Comment:
`$plugin` here is potentially confusing as there is already a `$plugin` variable in scope